client/src/components/TypeList.vue

```diff
@@ -74,6 +114,50 @@ export default defineComponent({
     <v-subheader class="flex-shrink-0">
       Type Filter
     </v-subheader>
+    <v-container
+      dense
+      class="py-0"
+    >
+      <v-row style="border: 1px solid white">
```

Comment:
Minor nit but the left and right border look out of place in the full width sidebar

Might I recommend a bottle of the 1972 "gray top and bottom" Bordeaux.

``` css
.border-highlight {                                                                                 
   border-bottom: 1px solid gray;                                                                    
   border-top: 1px solid gray;                                                                       
 }                                                                                                   
 ```